Crash report 3.9-rc-4: NPE in BackgroundDownloadWebImage

```
java.lang.RuntimeException: An error occured while executing doInBackground()
       at android.os.AsyncTask$3.done(AsyncTask.java:300)
       at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:355)
       at java.util.concurrent.FutureTask.setException(FutureTask.java:222)
       at java.util.concurrent.FutureTask.run(FutureTask.java:242)
       at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
       at java.lang.Thread.run(Thread.java:841)
Caused by: java.lang.NullPointerException: key == null || value == null
       at android.support.v4.util.LruCache.put(LruCache.java:133)
       at org.wordpress.android.ui.media.MediaUtils$BackgroundDownloadWebImage.doInBackground(MediaUtils.java:134)
       at org.wordpress.android.ui.media.MediaUtils$BackgroundDownloadWebImage.doInBackground(MediaUtils.java:118)
       at android.os.AsyncTask$2.call(AsyncTask.java:288)
       at java.util.concurrent.FutureTask.run(FutureTask.java:237)
       at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
       at java.lang.Thread.run(Thread.java:841)
```

Test the bitmap returned by `decodeStream`?

--- a/WordPress/src/main/java/org/wordpress/android/ui/media/MediaUtils.java
+++ b/WordPress/src/main/java/org/wordpress/android/ui/media/MediaUtils.java
@@ -21,7 +21,6 @@ import android.widget.ImageView;
 import org.wordpress.android.R;
 import org.wordpress.android.WordPress;
 import org.wordpress.android.WordPressDB;
-import org.wordpress.android.models.Blog;
 import org.wordpress.android.models.MediaFile;
 import org.wordpress.android.util.AppLog;
 import org.wordpress.android.util.AppLog.T;
@@ -131,7 +130,9 @@ public class MediaUtils {
                 if (bitmap == null) {
                     URL url = new URL(uri);
                     bitmap = BitmapFactory.decodeStream(url.openConnection().getInputStream());
-                    WordPress.getBitmapCache().put(uri, bitmap);
+                    if (bitmap != null) {
+                        WordPress.getBitmapCache().put(uri, bitmap);
+                    }
                 }
 
                 return bitmap;
@@ -145,7 +146,7 @@ public class MediaUtils {
         protected void onPostExecute(Bitmap result) {
             ImageView imageView = mReference.get();
 
-            if (imageView != null) {
+            if (imageView != null && result != null) {
                 if (imageView.getTag() == this) {
                     imageView.setImageBitmap(result);
                     fadeInImage(imageView, result);

